.github/workflows/python-test.yml

```diff
@@ -34,4 +39,4 @@ jobs:
         python -m pip install --upgrade pip
         pip install tox tox-gh-actions
     - name: Test with tox
-      run: tox
+      run: tox -e py
```

Comment:
You probably meant to select the Python version here, right? There is no `py` target.

Reply:
when `py` is provided as environment, `tox` will use the interpreter version that tox is executed with; which makes sense as the workflow environment is already using the designated interpreter.

unfortunately i can't find the relevant docs, but you can try it at home.

Reply:
Ah, sorry, I misread the (collapsed) CI output. It actually does what you say, just fails afterwards. Since you removed the pytest-mypy plugin and moved the `mypy` target out of the test jobs, it now no longer finds anything to test, and thus fails. The tests required the plugin to run.

https://github.com/lxml/lxml-stubs/runs/4264368265?check_suite_focus=true

Reply:
wouldn't it be sufficient to run `mypy` with each interpreter, or what is the reason to wrap it in pytest?

Reply:
If I'm not mistaken, removing the plugin requires the whole test suite be removed and rewritten from scratch.

Reply:
> wouldn't it be sufficient to run `mypy` with each interpreter, or what is the reason to wrap it in pytest?

It should really be enough to run mypy just once, not for each interpreter.

Reply:
sorry for the long delay. i re-added the plugin as test requirement.
and as a remark, each time i look into this, i find it exhausting to wrap my head around a test suite in a yaml file.